runtime/vm/CRIUHelpers.cpp

```diff
@@ -1880,7 +1878,6 @@ criuCheckpointJVMImpl(JNIEnv *env,
 			j9port_control(J9PORT_CTLDATA_SYSLOG_OPEN, 0);
 			setLogOptions(vm, syslogOptions);
 		}
-		j9mem_free_memory(syslogOptions);
```

Comment:
There is no usage of `syslogOptions` after this, I think only `j9mem_free_memory(syslogOptions)` at L1895 needs to be removed.

Reply:
I want to have a singluar call to `j9mem_free_memory(syslogOptions);` so we dont get into this situation again when code is refactored